### PR TITLE
Migrate parent pom [WIP]

### PR DESCRIPTION
**Pull request by etiennehomer**
Work in progress. First changes on powsybl-parent/powsybl-parent-ws/pom.xml

### CODE REVIEW

File: powsybl-parent/powsybl-parent-ws/pom.xml [hunk 1]
```diff
@@ -25,17 +25,32 @@
     <description>Powsybl Parent for Web Services</description>
 
     <properties>
+    
+    	<java.version>11</java.version>
+
```
Comment (ChamseddineBhd) [inline powsybl-parent/powsybl-parent-ws/pom.xml:30]: update the readMe file 
a default target java version (8) => a default target java version (11)

File: powsybl-parent/powsybl-parent-ws/pom.xml [hunk 2]
```diff
@@ -79,10 +94,18 @@
                 </plugin>
 
                 <!-- copied from spring-boot-starter-parent -->
+<!--                 <plugin> -->
```
Comment (ChamseddineBhd) [inline powsybl-parent/powsybl-parent-ws/pom.xml:97]: why this is commented ? remove it if it's useless

File: powsybl-parent/powsybl-parent-ws/pom.xml [hunk 3]
```diff
@@ -25,17 +25,32 @@
...
+    
+    	<java.version>11</java.version>
+    	
         <maven.jib.version>1.8.0</maven.jib.version>
         <maven.spring-boot.version>2.2.2.RELEASE</maven.spring-boot.version>
         <maven.git-commit-id.version>3.0.1</maven.git-commit-id.version>
+        
+        
+        <maven.clean.version>3.1.0</maven.clean.version>
+        <maven.compiler.version>3.8.0</maven.compiler.version>
```
Comment (ChamseddineBhd) [inline powsybl-parent/powsybl-parent-ws/pom.xml:37]: identation is not the same as <java.version>11</java.version> line

File: powsybl-parent/powsybl-parent-ws/pom.xml [hunk 4]
```diff
@@ -129,8 +152,63 @@
...
+                    <version>${maven.site.version}</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>${maven.surefire.version}</version>
+                </plugin>
             </plugins>
         </pluginManagement>
+        <plugins>
```
Comment (ChamseddineBhd) [inline powsybl-parent/powsybl-parent-ws/pom.xml:177]: i think even those plugins should be in the pluginManagment section ?
@jonenst is that right ?

File: powsybl-parent/powsybl-parent-ws/pom.xml [hunk 5]
```diff
@@ -129,8 +152,63 @@
...
+                            <consoleOutput>true</consoleOutput>
+                            <failsOnError>true</failsOnError>
+                            <includeTestSourceDirectory>true</includeTestSourceDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
```
Comment (ChamseddineBhd) [inline powsybl-parent/powsybl-parent-ws/pom.xml:199]: <version>${maven.spring-boot.version}</version> ?

File: powsybl-parent/powsybl-parent-ws/pom.xml [hunk 6]
```diff
@@ -25,17 +25,32 @@
...
+        <maven.compiler.version>3.8.0</maven.compiler.version>
+        <maven.install.version>2.5.2</maven.install.version>
+        <maven.jacoco.version>0.8.2</maven.jacoco.version>
+        <maven.resources.version>3.1.0</maven.resources.version>
+        <maven.site.version>3.7.1</maven.site.version>
+        <maven.surefire.version>2.22.1</maven.surefire.version>
+        <!-- copied from spring-boot-starter-parent -->
+        <maven.resource.delimiter>@</maven.resource.delimiter>
+        <maven.compiler.parameters>true</maven.compiler.parameters>
+        <springboot.version>2.2.2.RELEASE</springboot.version>
```
Comment (ChamseddineBhd) [inline powsybl-parent/powsybl-parent-ws/pom.xml:46]: same  as  <maven.spring-boot.version>2.2.2.RELEASE</maven.spring-boot.version>
So remove <maven.spring-boot.version>2.2.2.RELEASE</maven.spring-boot.version>

File: powsybl-parent/powsybl-parent-ws/pom.xml (hunk 5 above)
Comment (ChamseddineBhd) [reply powsybl-parent/powsybl-parent-ws/pom.xml:199]: or ${springboot.version}

File: powsybl-parent/powsybl-parent-ws/pom.xml (hunk 4 above)
Comment (jonenst) [reply powsybl-parent/powsybl-parent-ws/pom.xml:177]: checkstyle belongs in plugins not in pluginManagement.
spring-boot belongs in pluginManagement

But none of them should be here in this pom

Comment (etiennehomer) [reply powsybl-parent/powsybl-parent-ws/pom.xml:177]: They should be in powsybl-parent/pom.xml only ?

**Pull request closed without merging**